### PR TITLE
EOS-15360: motr status is not updated in Consul in case of failure

### PR DESCRIPTION
**Pull request by mssawant**
Solution:
Update Consul status in case of motr failure.

### CODE REVIEW

**Comment by knekrasov** [comment]
@mssawant Is this PR still relevant? Maybe it is time to merge it?

File: hax/hax/handler.py [hunk 1]
```diff
@@ -50,14 +51,38 @@ def __init__(self, q: Queue, motr: Motr):
...
+    def check_state_and_update(self, ha_states: List[HAState]) -> None:
+        for state in ha_states:
+            if state.status == ServiceHealth.FAILED:
+                m0status = m0HaProcessEvent.M0_CONF_HA_PROCESS_STOPPED
+                pevent = ConfHaProcess(chp_event=m0status,
+                                       chp_type=3,
+                                       chp_pid=0,
+                                       fid=state.fid)
+                self._update_process_status(pevent)
+                LOG.info('MANDAR fid: %s state: %s',
```
Comment (knekrasov) [inline hax/hax/handler.py:78]: I guess this is a debug logging

File: hax/hax/handler.py [hunk 2]
```diff
@@ -50,14 +51,38 @@ def __init__(self, q: Queue, motr: Motr):
...
+                         state.fid, m0status)
+
     def _do_work(self, q: Queue, motr: Motr):
         ffi = motr._ffi
         LOG.info('Handler thread has started')
         ffi.adopt_motr_thread()
+        self.statuses = self.consul.get_m0d_statuses()
```
Comment (knekrasov) [inline hax/hax/handler.py:85]: How `self.statuses` are used? What are they for?

File: hax/hax/types.py [hunk 3]
```diff
@@ -218,21 +218,22 @@ def __repr__(self):
 HAState = NamedTuple('HAState', [('fid', Fid), ('status', ServiceHealth)])
 
 
-class m0HaProcessEvent(Enum):
+class m0HaProcessEvent(IntEnum):
```
Comment (knekrasov) [inline hax/hax/types.py:221]: Why IntEnum is more suitable than Enum?

**Comment by mssawant** [comment]
@knekrasov, This problem is fixed by fd1b6fad, will abandon this patch as it is obsolete and no more required.
Thanks for bringing this up.

**Pull request closed without merging**